src/systemtest/resources/features/SourceChangeFlow.feature

```diff
@@ -10,9 +10,36 @@ Feature: Source Change system test
     And a jenkins job '"ActT1Job"' from '"src/systemtest/resources/JenkinsShellScripts/ActT1Script.txt"' is created with parameters: EVENT_ID,
     And a jenkins job '"ActSFCJob"' from '"src/systemtest/resources/JenkinsShellScripts/ActSFCScript.txt"' is created with parameters: EVENT_ID,
     And a jenkins job '"CLMJob"' from '"src/systemtest/resources/JenkinsShellScripts/CLMScript.txt"' is created with parameters: EVENT_ID,
+    And a jenkins job '"FlowCompleteJob"' from '"src/systemtest/resources/JenkinsShellScripts/FlowCompleteScript.txt"' is created with parameters: ,
 
-    ### Add Subscription to EI ###
+    ### Add CLMSubscription to EI ###
+    Given subscription object "CLMSubscription" is created which will trigger "CLMJob" with parameters
+    When notification with key "EVENT_ID" and value "id" is added to "CLMSubscription"
+    And condition with jmespath "submission.gitIdentifier.repoName=='my-repo'" is added to "CLMSubscription"
+    And condition with jmespath "submission.submitter.email=='jane@example.com'" is added to "CLMSubscription"
+    Then we send the "CLMSubscription" to eiffel intelligence for creation.
 
+    ### Add ActT1Subscription to EI ###
+    Given subscription object "ActT1Subscription" is created which will trigger "ActT1Job" with parameters
+    When notification with key "EVENT_ID" and value "submission.confidenceLevels[0].eventId" is added to "ActT1Subscription"
+    And condition with jmespath "submission.confidenceLevels[0].value=='SUCCESS'" is added to "ActT1Subscription"
+    Then we send the "ActT1Subscription" to eiffel intelligence for creation.
+
+    ### Add ActSFCSubscription to EI ###
+    Given subscription object "ActSFCSubscription" is created which will trigger "ActSFCJob" with parameters
+    When notification with key "EVENT_ID" and value "causedActivities[0].triggeredEventId" is added to "ActSFCSubscription"
+    And condition with jmespath "causedActivities[0].name=='TriggeredOnSourceChangeSubmitted'" is added to "ActSFCSubscription"
```

Comment:
do not use index. check on name in array like this. causedActivities[?name=='TriggeredSourceChangeSubmitted]

Reply:
Updated conditions to check on names instead of indexes.